examples/index.html

```diff
@@ -89,8 +90,8 @@
         Minified version size: ~4kb
     </p>
     <br>
-    <a class="remodal-cancel" href="#">Cancel</a>
-    <a class="remodal-confirm" href="#">OK</a>
+    <span data-remodal-action="cancel" class="remodal-cancel">Cancel</span>
```

Comment:
Well, i didn't think, that it is necessary for examples, but maybe you're right. I've changed tags to `button`